src/ui/gui.css

```diff
@@ -286,8 +286,17 @@ select.center {
 
 .die_overlay {
     position:relative;
-    font-size: 24px;
-    top: 13px;
+    font-size: 21px;
+    top: 14px;
+}
+
+/* Added slightly different vertical spacing for Firefox only */
+@-moz-document url-prefix() {
+    .die_overlay{
+        position:relative;
+        font-size: 21px;
+        top: 17px;
+    }
```

Comment:
Syntax question: do you actually have to redefine the things which are unchanged (position, font-size) in this block, or could you just define the thing which is an override?  If the latter is supported, it would be clearer IMO.